packages/esm-patient-chart-app/src/config-schema.ts

```diff
@@ -22,6 +22,19 @@ export const esmPatientChartSchema = {
     _description: 'The UUID of the visit type to be used for the automatically created offline visits.',
     _default: 'a22733fa-3501-4020-a520-da024eeff088',
   },
+  navGroup: {
```

Comment:
I think this makes it clear that we need extension-specific schemas. They can inherit from module schemas (which is desirable for things like concepts). But this stuff just doesn't make sense in the module schema.

Reply:
@brandones I'm curious to know whether this supports multiple groups. In other words is possible to define multiple groups using the configuration approach?

Reply:
It does! You can add the same extension multiple times and configure it using the `#id` pattern:

```json
"extensions": {
  "add": ["nav-group#hiv", "nav-group#heart"],
  "configure": {
    "nav-group#hiv": {
     "navGroup": { "title": "HIV", slotName: "hiv-nav-group-slot" }
    },
    "nav-group#heart": {
      "navGroup": { "title": "Cardiology", "slotName": "heart-nav-group-slot" }
    }
  }
}
```